packages/next-codemod/transforms/built-in-next-font.ts

```diff
@@ -35,8 +38,13 @@ export default function transformer(
       source: { value: '@next/font/local' },
     })
     .forEach((fontImport) => {
+      hasChanges = true
       fontImport.node.source = j.stringLiteral('next/font/local')
     })
 
-  return root.toSource(options)
+  if (hasChanges) {
+    return root.toSource(options)
+  }
+
+  return file.source
```

Comment:
```suggestion
  return hasChanges ? root.toSource(options) : file.source
```

You can do this in one line